pkg/printers/tableprinter.go

```diff
@@ -117,6 +128,8 @@ func (h *HumanReadablePrinter) PrintObj(obj runtime.Object, output io.Writer) er
 			h.printedHeaders = true
 		}
 
+		// TODO(seans3): Remove the following decorateTable call. Table modification
+		// (and creation) should only happen in table generation (tablegenerator.go).
```

Comment:
hmm... actually, looking at what decorate table does, it is adding generic data based on client-side options (print namespace, include kind, include labels, etc). Those print options aren't propagated to the server, so there's no way the server could know whether or not to do those things currently. I think we should actually remove the decorate table call from the server-side table generation

Reply:
Got it. That makes sense. I moved the TODO to `tablegenerator.go`. I intend to address the `decorateTable()` issue in the next PR.